packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -104,5 +109,62 @@ describe('plugin-conversation', () => {
         assert.equal(url, convoUrl);
       });
     });
+    describe('#resendPostActivity', () => {
+      const testConvo = {test: 'convo'};
+
+      it('should send post activity', () => {
+        const activity = {
+          actor: {
+            objectType: 'person',
+            id: 'id'
+          },
+          object: {
+            displayName: 'updateKey',
+            content: 'updateKey',
+            objectType: 'conversation'
+          },
+          target: {
+            //  mock encryption service url
+            defaultActivityEncryptionKeyUrl: `${process.env.ENCRYPTION_SERVICE_URL || 'https://encryption-a.wbx2.com'}/encryption/api/v1/keys/8a7d3d78-ce75-48aa-a943-2e8acf63fbc9`
```

Comment:
If the request is mocked, do we need the env var injection here?

Reply:
Not really. will remove